plugins/wazo-fanvil/common/common.py

```diff
@@ -200,6 +204,7 @@ def configure_common(self, raw_config):
             tpl = self._tpl_helper.get_template(f'common/{tpl_filename}')
             dst = os.path.join(self._tftpboot_dir, filename)
             raw_config['XX_fw_filename'] = fw_filename
+            raw_config['XX_model_info'] = _
```

Comment:
Why `_`?

Reply:
to add phone set model dynamically in plugins `/wazo-fanvil/common/templates/common/model-v6x.tpl` (for exemple) firmware download info file

Reply:
This is not clear nor good practice. Please define explicitly what the value should be. I guess you want it to be  `fw_filename`. In that case write `fw_filename`. `_` is just an ugly hack that Python provides that should be used when you want to discard a value or use the previous value in an *interactive session* (when you get a REPL). This is not an interactive session.
Thank you!

Reply:
@bloom1 fix done